Dockerfile

```diff
@@ -1,76 +1,56 @@
-# Create common base stage
-FROM python:3.6-slim as base
-
-WORKDIR /build
-
-# Create virtualenv to isolate builds
-RUN python -m venv /build
-
-# Install common libraries
-RUN apt-get update -qq \
- && apt-get install -y --no-install-recommends \
-    # required by psycopg2 at build and runtime
-    libpq-dev \
-     # required for health check
-    curl \
- && apt-get autoremove -y
-
-# Make sure we use the virtualenv
-ENV PATH="/build/bin:$PATH"
-
-# Stage to build and install everything
-FROM base as builder
-
-WORKDIR /src
-
-# Install all required build libraries
-RUN apt-get update -qq \
- && apt-get install -y --no-install-recommends \
-    build-essential \
-    wget \
-    openssh-client \
-    graphviz-dev \
-    pkg-config \
-    git-core \
-    openssl \
-    libssl-dev \
-    libffi6 \
-    libffi-dev \
-    libpng-dev
-
-# Copy only what we really need
-COPY README.md .
-COPY setup.py .
-COPY setup.cfg .
-COPY MANIFEST.in .
-COPY requirements.txt .
-COPY LICENSE.txt .
-
-# Install dependencies
-RUN pip install -U pip && pip install --no-cache-dir -r requirements.txt
-
-# Install Rasa as package
-COPY rasa ./rasa
-RUN pip install .[sql]
-
-# Runtime stage which uses the virtualenv which we built in the previous stage
-FROM base AS runner
-
-# Copy virtualenv from previous stage
-COPY --from=builder /build /build
-
-WORKDIR /app
+FROM python:3.6-slim as python_builder
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+  build-essential \
+  wget \
+  openssh-client \
+  graphviz-dev \
+  pkg-config \
+  git-core \
+  openssl \
+  libssl-dev \
+  libffi6 \
+  libffi-dev \
+  libpng-dev \
+  # required by psycopg2 at build and runtime
+  libpq-dev \
```

Comment:
we also need this during runtime! (however, we can dump curl)

Reply:
Are you sure about that? Current dockerfiles also don't install it: https://github.com/RasaHQ/rasa/blob/master/Dockerfile

(or maybe I got wrong what you're saying)

Reply:
it's installed in the `base` stage and the `base` stage is then re-used in the `runner` stage https://github.com/RasaHQ/rasa/blob/4234433b438d6ed4c24d66d0b81b396d91f8745f/Dockerfile#L57

Reply:
Oh interesting. Ok, will check it

Reply:
done!